src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -221,15 +221,15 @@ private static ProcessModuleCollection GetModules(int processId, bool firstModul
             }
         }
 
-        private static void EnumProcessModulesUntilSuccess(SafeProcessHandle processHandle, IntPtr[]? modules, int size, out int needed)
+        private static void EnumProcessModulesUntilSuccess(SafeProcessHandle processHandle, IntPtr[]? modules, int size, out int needed, int filterFlag)
```

Comment:
nit: if `Interop.Kernel32.LIST_MODULES_ALL` is the only value we use, you could avoid adding the new parameter to `EnumProcessModulesUntilSuccess` and just always pass `Interop.Kernel32.LIST_MODULES_ALL` to `EnumProcessModulesEx`

Reply:
Sorry, didn't wanna break any of the existing patterns.